tests/unit/amo/sagas/testReviews.js

```diff
@@ -3,13 +3,19 @@ import SagaTester from 'redux-saga-tester';
 import * as reviewsApi from 'amo/api/reviews';
 import {
   fetchReviews,
+  flagReview,
   hideReplyToReviewForm,
   setAddonReviews,
   sendReplyToReview,
   setReview,
   setReviewReply,
+  setReviewWasFlagged,
 } from 'amo/actions/reviews';
-import { SET_ADDON_REVIEWS } from 'amo/constants';
+import {
+  REVIEW_FLAG_REASON_OTHER,
+  REVIEW_FLAG_REASON_SPAM,
```

Comment:
Oh wait, I see, the reasons vary. So maybe ignore my comments about flagging as abusive. But I still think "report" is a better verb than "flag".